content/management/changelog/index.mdx

```diff
@@ -49,10 +49,23 @@ Since the migration is still ongoing, different parts will be successively moved
 
 ### Webhooks
 
-- The [**Unregister Webhook**](/management/configuration-api/v3.3/#unregister-webhook) method had the `webhook_id` parameter renamed to `id`.
+- The [**Unregister Webhook**](/management/configuration-api/v3.3/#unregister-webhook) method:
+  - had the `webhook_id` parameter renamed to `id`.
+  - had required scope changed to `webhooks.configuration:rw`.
+  - has a new parameter `owner_client_id`. It's required when authorizing with Personal Access Token; otherwise ignored.
+- The [**List Webhook Names**](/management/configuration-api/v3.3/#list-webhook-names) method was added.
```

Comment:
I think we should move it below the entry on the Register Webhook method. Otherwise, the reader might miss it (it's squeezed in between two larger sections.)